src/Nri/Ui/Button/V7.elm

```diff
@@ -0,0 +1,808 @@
+module Nri.Ui.Button.V7 exposing
```

Comment:
Actual Diff: 

```
< module Nri.Ui.Button.V6 exposing
---
> module Nri.Ui.Button.V7 exposing
10c10
< # Changes from V5:
---
> # Changes from V6:
12c12,13
<   - Update version of Nri.Ui.Icon to V4
---
>   - WidthUnbounded button displays inlineFlex, so that it doesn't accidentally take up the entire width
>   - Add WidthFillContainer, for when we intend to take up the full container
85a87
>     | WidthFillContainer
559d560
<         , Css.display Css.inlineBlock
573c574
<         , Css.displayFlex
---
>         , Css.display Css.inlineFlex
758a760,766
>                 WidthFillContainer ->
>                     [ Css.paddingLeft (Css.px 16)
>                     , Css.paddingRight (Css.px 16)
>                     , Css.minWidth (Css.px config.minWidth)
>                     , Css.width (Css.pct 100)
>                     ]
> 
810c818
<     "Nri-Ui-Button-V6-" ++ suffix
---
>     "Nri-Ui-Button-V7-" ++ suffix
```

Reply:
Thanks for this diff!